src/common/constants/index.ts

```diff
@@ -48,3 +48,9 @@ export const MESSAGES = {
   REJECT: "신청을 거절하시겠어요?",
   NOTIFICATION: "알림이 없습니다.",
 } as const;
+
+export const ERROR_MESSAGES = {
+  EMAIL: "잘못된 이메일입니다.",
+  PASSWORD: "8자 이상 입력해 주세요.",
+  PASSWORDVALID: "비밀번호가 일치하지 않습니다.",
```

Comment:
key값이랑 내용이 일치한다고 보기엔 어려울 것 같네요~
좋은 이름으로 작성된 건 아닌 것 같지만 이런식으로 어떤 에러인지 의미가 드러나야 실제로 ERROR_MESSAGES라는 객체를 사용하기 편리할 것 같네요~
```suggestion
export const ERROR_MESSAGES = {
  INVALID_EMAIL: "잘못된 이메일입니다.",
  SHORT_PASSWORD: "8자 이상 입력해 주세요.",
  INCORRECT_PASSWORD: "비밀번호가 일치하지 않습니다.",
```